src/core/server/server.ts

```diff
@@ -550,4 +556,33 @@ export class Server {
       },
     });
   }
+
+  /**
+   * Reports the new and legacy KIBANA_STARTED_EVENT.
+   * @param analyticsStart The {@link AnalyticsServiceStart}.
+   * @private
+   */
+  private reportKibanaStartedEvents(analyticsStart: AnalyticsServiceStart) {
+    // Report the legacy KIBANA_STARTED_EVENT.
+    analyticsStart.reportEvent(KIBANA_STARTED_EVENT, { uptime_per_step: this.uptimePerStep });
+
+    const ups = this.uptimePerStep;
+
+    const toMs = (sec: number) => Math.round(sec * 100);
```

Comment:
I just realized about this: We need to multiply by 1000 to parse seconds to milliseconds, don't we?